app/assets/javascripts/components/action.js.jsx

```diff
@@ -1,7 +1,9 @@
 var Action = React.createClass({
   render: function () {
     return (
-      <a className={this.props.data.className} onClick={this.props.data.action}>{this.props.data.title}</a>
+      <span style={{ paddingRight: '4px' }}>
```

Comment:
Looks like this is adding some extra spacing to the instructor buttons?

<img src="https://user-images.githubusercontent.com/5882053/100363941-6daf3e80-2fcb-11eb-8127-801f2f56e038.png" width="400" />

Reply:
Oops, sorry, that actually wasn't needed! I had added this part to address some formatting issues in the request, but it seems like this particular addition was not necessary to fix those issues. l'll push my changes again!